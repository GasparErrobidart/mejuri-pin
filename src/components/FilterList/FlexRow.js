import styled              from 'styled-components'

const FlexRow = styled.div`
  display: flex;
  & > *:first-child{
    flex-grow:1;
  }
`

export default FlexRow
