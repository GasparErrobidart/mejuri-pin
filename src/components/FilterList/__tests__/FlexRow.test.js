import '@testing-library/jest-dom'
import 'jest-styled-components'

import React                 from 'react'
import {ThemeProvider}       from 'styled-components'
import { cleanup }           from '@testing-library/react'
import renderer              from 'react-test-renderer'

import theme                 from '../../GlobalStyle/themes/mejuri-pin'

import FlexRow   from '../FlexRow'


afterEach(cleanup)

test('Renders "FlexRow" component', () => {
  const testText        = "FlexRow Test Text"
  const component       = renderer.create(
    <ThemeProvider theme={theme}>
      <FlexRow>{testText}</FlexRow>
    </ThemeProvider>);
  const tree            = component.toJSON()
  expect(tree).toMatchSnapshot();
});
