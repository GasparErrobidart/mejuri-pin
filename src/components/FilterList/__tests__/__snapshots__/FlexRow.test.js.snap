// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders "FlexRow" component 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c0 > *:first-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

<div
  className="c0"
>
  FlexRow Test Text
</div>
`;
